scripts/load-env

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+# this script is designed to be sourced from other scripts
+
+# function to load and set out environment config
+load-env() {
```

Comment:
This is a nice bit of refactoring. It means that for loop is now self-documenting.

Reply:
Right, it also means if I edit the config, I can reload it into my current shell with `load-env`, which I've wanted in the past